fuel_test/prep_env.sh

```diff
@@ -154,7 +179,11 @@ function wait_for_nailgun {
 }
 
 echo "Restoring Fuel Master.."
-restore_fm "$XS_HOST" "$FM_NAME" "$FM_SNAPSHOT"
+if [ -n "$FM_SNAPSHOT" ]; then
+	restore_fm_snapshot "$XS_HOST" "$FM_NAME" "$FM_SNAPSHOT"
```

Comment:
Should test here if the snapshot exists on the xenServer host, and fallback to trying to import the XVA if it does not.